Recrutify/Recrutify.DataAccess/Models/EmailRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Recrutify.DataAccess.Models
+{
+    public class EmailRequest
```

Comment:
you can use standard class instead of  it https://docs.microsoft.com/en-us/dotnet/api/system.net.mail.mailmessage?view=net-5.0